### DISCUSSION TITLE
algerians tv

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#221] [Q&A]
can you extract links of this channels
https://itvepg2.at.dz/iptvepg/frame1039/tv.jsp?ix=1&index=2

**Comment by BellezaEmporium** [comment]
Got nothing in this, it asks for a session rebuild and then just says nothing.

P.S = are you sure you're in the correct section and not asking for something to be added in the [iptv-org/epg](https://github.com/iptv-org/epg) section ?

**Comment by ARAB-IPTV** [comment]
yes im sure , visite the website , you will find the live transmition of the algerians tv

**Comment by ARAB-IPTV** [comment]
this is the link of the algerian national tv 1
https://itv033022.at.dz:6630/001/2/ch00000090990000001035/manifest.mpd?JITPDRMType=Widevine&version=v1.0&IASHttpSessionId=RR1690620211022122118000038&programid=ch00000000000000001021&virtualDomain=001.live_hls.zte.com&recommendtype=0&ctype=4&profilecode=&JITPMediaType=DASH&boid=001&terminalflag=4&contentid=ch00000000000000001021&BreakPoint=0&NeedJITP=1&AuthInfo=%2Baea96bf8qANngDFqDsD5BF8FUZQuIhUmDvxGQmCGJJOXBI6LYgKUGgTAW9oMDl1c6l%2BmK6q%2FE%2BfMu852MXd0w%3D%3D&stbid=&userid=guestpc&videoid=ch00000090990000001035

**Comment by BellezaEmporium** [comment]
FR : Je n'ai aucun accès au site malheureusement, il bloque lors de la reconstruction des cookies de session.

EN : I got no access to the website unfortunately, it blocks when reconstructing session cookies.

**Comment by ARAB-IPTV** [comment]
ok merci , je pense que le site bloqué en dors l'algérie :(

**Comment by BellezaEmporium** [comment]
Au vu des commentaires sur le code source du site + le lien partagé pour la chaîne, j'ai bien l'impression que ce sont des Chinois derrière tout ceci. Le domaine virtuel hls.zte.com est un serveur chinois. C'est bien la première fois que je vois des chaînes algériennes en dehors de la diaspora européenne.

**Comment by ARAB-IPTV** [comment]
parceque algerie telecom le fournisseur officielle d'internet en algerie travaille exclusivement avec les chionois , presque tout les produits le societé est chinois " zte , huawei "

**Comment by BellezaEmporium** [comment]
Intéressant, ceci explique beaucoup de choses. Je vais voir ça.

**Comment by BellezaEmporium** [comment]
J'ai réussi à accéder au site je ne sais comment. J'ai trouvé comment les liens étaient faits, et il y a aussi des liens HLS de disponibles, qui pourraient donc fonctionner avec VLC. Cependant, il m'est impossible d'accéder à quoi que ce soit, ça charge sans rien donner en retour.

**Comment by BellezaEmporium** [comment]
Pour exemple, ce lien donnera un JSON contenant deux liens : un HTTP (sous adresse IP), un autre sous HTTPS (avec un serveur sous identifiant, géré par le site) : https://itvepg2.at.dz/iptvepg/frame1039/action/sdk_gettvplayurl.jsp?channelcode=ch00000000000000001021&columncode=0101&definition=1&mediaservice=2&authid=348178&mediaservice=2&urlredirect=1&needjitp=1&jitpmediatype=HLS&jitpdrmtype=&playtype=0&timestamp=1634915029142

**Comment by ARAB-IPTV** [comment]
je trouve des liens marche bien sur VLC mais je sais pas est ce que ca dure longtemps ou nn , les lien au format "mpd"
